src/MudBlazorWebTemplate/.template.config/localize/templatestrings.pt-BR.json

```diff
@@ -0,0 +1,44 @@
+  "symbols/InteractivityPlatform/displayName": "_Tipo de interatividade",
+  "symbols/InteractivityPlatform/description": "Escolhe qual plataforma de hospedagem usar com os componentes interativos",
+  "symbols/InteractivityPlatform/choices/None/displayName": "Nenhum",
+  "symbols/InteractivityPlatform/choices/None/description": "Sem interatividade (somente renderização do servidor estático)",
+  "symbols/InteractivityPlatform/choices/Server/displayName": "Servidor",
+  "symbols/InteractivityPlatform/choices/Server/description": "É executado no servidor",
+  "symbols/InteractivityPlatform/choices/WebAssembly/displayName": "WebAssembly",
+  "symbols/InteractivityPlatform/choices/WebAssembly/description": "É executado no navegador usando o WebAssembly",
+  "symbols/InteractivityPlatform/choices/Auto/displayName": "Automático (Servidor e WebAssembly)",
+  "symbols/InteractivityPlatform/choices/Auto/description": "Usa Servidor ao baixar os ativos WebAssembly e usa o WebAssembly",
+  "symbols/InteractivityLocation/displayName": "_Interatividade local",
+  "symbols/InteractivityLocation/description": "Escolhe quais componentes terão a renderização interativa habilitada",
+  "symbols/InteractivityLocation/choices/InteractivePerPage/displayName": "Por página/componente",
+  "symbols/InteractivityLocation/choices/InteractivePerPage/description": "A interatividade é aplicada por página ou por componente",
+  "symbols/InteractivityLocation/choices/InteractiveGlobal/displayName": "Global",
+  "symbols/InteractivityLocation/choices/InteractiveGlobal/description": "A interatividade é aplicada no nível raiz",
+  "symbols/IncludeSampleContent/displayName": "_Incluir páginas de amostra",
+  "symbols/IncludeSampleContent/description": "Configura se deseja adicionar páginas de amostra e estilo para demonstrar padrões de uso básicos.",
+  "symbols/Empty/description": "Configura a omissão de páginas de amostra e estilo que demonstram padrões básicos de uso.",
+  "symbols/auth/choices/None/description": "Sem autenticação",
+  "symbols/auth/choices/Individual/description": "Autenticação individual",
+  "symbols/auth/description": "O tipo de autenticação a ser usado",
+  "symbols/UseLocalDB/description": "Se deve usar LocalDB em vez de SQLite. Esta opção só se aplica se --auth Individual for especificado.",
+  "symbols/AllInteractive/displayName": "_Habilitar renderização interativa globalmente no site",
+  "symbols/AllInteractive/description": "Configura se todas as páginas devem ser interativas aplicando um modo de renderização interativo no nível superior. Se for falso, as páginas usarão a renderização do servidor estático por padrão e poderão ser marcadas como interativas por página ou por componente.",
+  "symbols/NoHttps/description": "Se deve desligar o HTTPS. Essa opção só se aplica se Individual não for usado para --auth.",
```

Comment:
Desativar HTTPS. Essa opção só se aplica se Individual não for usado para --auth.